packages/desktop/src/main/main.ts

```diff
@@ -383,6 +392,13 @@ app.on('ready', async () => {
     mainWindow.webContents.send('force-save-state')
   })
 
+  splash.once('close', e => {
+    e.preventDefault()
+    log('Closing window')
+    mainWindow.webContents.send('force-save-state')
```

Comment:
I think it's not necessary here because there won't be any changes in the store yet, am I right?

Reply:
ye but it's triggering minWindow.close on `ipcMain.on('state-saved',...`